From 5018939a4e526250ded2053adb9c4f286e497077 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mat=C4=9Bj=20T=C3=BD=C4=8D?= <matyc@redhat.com>
Date: Tue, 8 Jan 2019 18:11:14 +0100
Subject: [PATCH] Remove the sshd_disable_rhosts_rsa rule from RHEL8 and Fedora
 profiles.

The rule references a sshd configuration option that is not relevant for up-to-date RHEL>=7 systems.
---
 fedora/profiles/ospp.profile | 1 -
 rhel8/profiles/hipaa.profile | 1 -
 rhel8/profiles/ospp.profile  | 1 -
 3 files changed, 3 deletions(-)

diff --git a/fedora/profiles/ospp.profile b/fedora/profiles/ospp.profile
index f6c3023e6b..4389b6d438 100644
--- a/fedora/profiles/ospp.profile
+++ b/fedora/profiles/ospp.profile
@@ -72,7 +72,6 @@ selections:
     - disable_host_auth
     - sshd_disable_gssapi_auth
     - sshd_disable_kerb_auth
-    - sshd_disable_rhosts_rsa
     - sshd_disable_rhosts
     - sshd_disable_user_known_hosts
     - var_accounts_passwords_pam_faillock_deny=3
diff --git a/rhel8/profiles/hipaa.profile b/rhel8/profiles/hipaa.profile
index feb98007cf..48fd03b46d 100644
--- a/rhel8/profiles/hipaa.profile
+++ b/rhel8/profiles/hipaa.profile
@@ -57,7 +57,6 @@ selections:
     - sshd_disable_compression
     - sshd_disable_gssapi_auth
     - sshd_disable_kerb_auth
-    - sshd_disable_rhosts_rsa
     - sshd_disable_rhosts
     - sshd_disable_user_known_hosts
     - sshd_do_not_permit_user_env
diff --git a/rhel8/profiles/ospp.profile b/rhel8/profiles/ospp.profile
index 9ecd2e90e5..9480206d89 100644
--- a/rhel8/profiles/ospp.profile
+++ b/rhel8/profiles/ospp.profile
@@ -67,7 +67,6 @@ selections:
     - disable_host_auth
     - sshd_disable_gssapi_auth
     - sshd_disable_kerb_auth
-    - sshd_disable_rhosts_rsa
     - sshd_disable_rhosts
     - sshd_disable_user_known_hosts
     - var_accounts_passwords_pam_faillock_deny=3
